README.md

```diff
@@ -11,20 +11,28 @@ In `mix.exs`, add the ExMachina dependency:
 
 ```elixir
 def deps do
-  [{:ex_machina, "~> 0.4"}]
+  [
+    {:ex_machina, "~> 0.4", only: "test"}
+  ]
 end
 ```
 
-Add `:ex_machina` to your application list:
+Add `:ex_machina` to your application list and also make sure that the `test/support` directory (where you'll put your factory definitions) will be compiled:
 
 ```elixir
 def application do
-  [applications: app_list(Mix.env)]
+  [
+    applications: app_list(Mix.env),
+    elixirc_paths: elixirc_paths(Mix.env)
```

Comment:
I believe that the `elixirc_paths` are set in the `project`. Does this also work?